ZenPacks/zenoss/HttpMonitor/datasources/HttpMonitorDataSource.py

```diff
@@ -1,157 +1,208 @@
 ##############################################################################
-# 
-# Copyright (C) Zenoss, Inc. 2008, all rights reserved.
-# 
+#
+# Copyright (C) Zenoss, Inc. 2018, all rights reserved.
+#
 # This content is made available according to terms specified in
 # License.zenoss under the directory where your Zenoss product is installed.
-# 
+#
 ##############################################################################
 
 
-__doc__='''HttpMonitorDataSource.py
+__doc__ = '''HttpMonitorDataSource.py
```

Comment:
FYI, the `__doc__ = ` portion of this statement is not necessary.  Python will automatically set the first unassigned string in a module as the docstring for that module.